src/Makefile.am

```diff
@@ -123,6 +123,7 @@ BITCOIN_CORE_H = \
   banman.h \
   base58.h \
   bech32.h \
+  bignum.h \
```

Comment:
nit: this should only be [here](https://github.com/vertcoin-project/vertcoin-core/pull/248/files#diff-4cb884d03ebb901069e4ee5de5d02538c40dd9b39919c615d8eaa9d364bbbd77R126)

Reply:
Removed the addition of bignum.h in this 'branding' commit.